server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -521,6 +522,12 @@ private DivTag administrativeNameField(String adminName, boolean editExistingQue
         .setId("question-name-input")
         .setFieldName(QUESTION_NAME_FIELD)
         .setLabelText("Administrative identifier. This value can't be changed later")
+        .setToolTipText(
+            "This is the name with which the question will be identified in the API export.  "
```

Comment:
suggestion (optional): "This will be used to identify questions in the API and CSV export. It will be formatted..."

reason: it's slightly more casual language

Reply:
Good idea.  I like that better.  Will change the wording.